src/composer.json

```diff
@@ -11,7 +11,9 @@
         "laravel/framework": "^10.10",
         "laravel/sanctum": "^3.3",
         "laravel/tinker": "^2.8",
-        "spatie/laravel-permission": "^6.1"
+        "mxl/laravel-job": "^1.5",
+        "spatie/laravel-permission": "^6.1",
+      "ext-curl": "*"
```

Comment:
Could you set the tabs/indents to 2 instead of 4?
Pretty please? 👉🏻 👈🏻

Reply:
hoe?

Reply:
Hebben we geen .editorconfig?

Reply:
done